spec/src/main/asciidoc/release_notes.asciidoc

```diff
@@ -79,6 +80,35 @@ A full list of changes may be found on the link:https://github.com/eclipse/micro
 === Other changes
 - Clarify failOn() on CircuitBreaker and Fallback (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/240[#240])
 - Circuit Breaker - clarify how requestVolumeThreshold() and rolling window work (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/342[#342])
-- other smaller fixes (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/341[#341]) (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/252[#252]) (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/306[#306]) 
+- other smaller fixes (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/341[#341]) (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/252[#252]) (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/306[#306])
 
+[[release_notes_21]]
+== Release Notes for MicroProfile Fault Tolerance 2.1
 
+This release is a minor release of Fault Tolerance.
+It includes a number of new features and clarifications, as well as TCK improvements.
+The following changes occurred in the 2.1 release, compared to 2.0.
+
+A full list of changes may be found on the link:https://github.com/eclipse/microprofile-fault-tolerance/milestone/8[MicroProfile Fault Tolerance 2.1 Milestone]
+
+=== API/SPI Changes
+
+- The `Retry.retryOn` and `abortOn` attributes no longer ignore `Throwable.class` (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/449[#449]).
+- Added `CircuitBreaker.skipOn` (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/418[#418]).
+- Added `Fallback.applyOn` and `skipOn` (link:https://github.com/eclipse/microprofile-fault-tolerance/issues/417[#417]).
+
+=== Functional Changes
```

Comment:
I don't quite understand the difference between "Functional Changes" and "Specification Changes", so I'm keeping this section empty for now.

Reply:
This means the implemenation impact.

Reply:
In this release, the functional changes are same as spec changes. Just say: Implement all spec changes.

Reply:
I've looked at the previous versions in this file, and "Functional Changes" always contain a mix of lines from other sections. This makes zero sense to me, I'd frankly much rather remove "Functional Changes" for good.

Reply:
ok. Let's remove it.